CSharp/BatchExplorer/Messages/NodeSchedulingMessage.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.BatchExplorer.Messages
+{
+    public enum SchedulingType
```

Comment:
"SchedulingType" is a new notion that doesn't map to the API. Wouldn't two messages, NodeEnableSchedulingMessage and NodeDisableSchedulingMessage, map more accurately to the two distinct REST operations/client methods?